CHANGELOG.md

```diff
@@ -6,6 +6,128 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0] - June, 27, 2022
+
+### Design Tokens
+#### Breaking Changes
+#### Changed
+- [Typography] Typography tokens to match hel.fi new visual identity
```

Comment:
This is probably too vague. If a developer only reads this it can be hard to understand what has been changed.

Maybe copy this from the design kit notes?

Added:
> - [Typography] New XXL heading size (64px)
> - [Typography] New XL-Mobile heading size (40px)

Changed:
> - [Typography] Made changes to HDS heading font sizes to match Hel.fi styles
>   - XL from 52 to 48px
>   - L from 36 to 32px
> - [Typography] Line-height tokens are no longer used for headings. All headings now have their own line-height value. Use helper CSS classes from Core to easily follow the new values.

Reply:
- Fix design tokens changelog